api/test/tests/controllers/participant.js

```diff
@@ -81,4 +97,32 @@ describe( 'controllers/participant', () => {
       }
     });
   });
+  describe( "#getMeetings", () => {
+    it( "should return meetings", async() => {
+
+      const res = await Meeting.insertMany( [ meeting1, meeting2 ] );
+
+      let participants = new Array();
+
+      res.forEach( ( meeting ) => {
+        participants.push(
+          { email: "jack@aol.com", meeting_id: meeting._id }
+        );
+      });
+
+      await Participant.insertMany( participants );
+
+      try{
+        const resMain = await client.post(
+          "participant/getmeetings", { email: "jack@aol.com" }
+        );
+        assert( resMain.data[1].name === "meeting2", "found Meetings" );
```

Comment:
To make sure that everything we want comes back in each meeting we can use `chai.assert.deepInclude` like this:

```js
chai.assert.deepInclude(
  resMain.data[ 0 ],
  {
    ...meeting1,       
    owner_id: meeting1.owner_id.toString()
  }
);
```

We have to convert the `owner_id` to a string because its an `ObjectID`.